partial/Content.php

```diff
@@ -16,10 +16,7 @@
     <div class="columns" id="bodyContent">
         <div class="column is-full" id="bodyContent">
             <?php
-            ob_start();
```

Comment:
These are not needed as you are already echoing to the buffer. Plus, they can interfere with existing buffers that might be used during skin rendering.

Reply:
@jdlrobson Hi, Recently I upgraded mediawiki to 1.39 and got this error (seems only occur when upload file or move a page):
```
[Sat Oct 07 10:19:01.152356 2023] [php7:notice] [pid 911074] [client 58.48.170.242:21520] PHP Notice:  Undefined variable: out in /var/www/wiki/skins/Pure/partial/Content.php on line 22, referer: https://wiki.riguz.com/index.php?title=Special:MovePage&action=submit
[Sat Oct 07 10:20:04.832883 2023] [php7:notice] [pid 905824] [client 51.222.253.7:63816] PHP Notice:  Undefined variable: out in /var/www/wiki/skins/Pure/partial/Content.php on line 22
[Sat Oct 07 10:20:49.720975 2023] [php7:notice] [pid 905822] [client 47.128.55.20:57012] PHP Notice:  Undefined variable: out in /var/www/wiki/skins/Pure/partial/Content.php on line 22
```
It works after I revert the following changes:

```diff
- ob_start();
   $this->html('bodytext');
- $out = ob_get_contents();
- ob_end_clean();
   $markers = array("<h1", "<h2", "<h3");
   $tags = array('<h1 class="title is-4"', '<h2 class="title is-5"', '<h3 class="title is-6"');
   $body = str_replace($markers, $tags, $out);
```
How should I fix this? I cannot remember how I got this skin work... it looks very unfamiliar to me :p